### ISSUE TITLE
Node download on s390x Linux on Z platform is not supported

### ISSUE THREAD

**Issue by mengwan9** [srs/gradle-node-plugin#379]
Hi, 

We are building our project with "com.moowork.node" plugin.  However, currently gradle node plugin can't recognize the platform s390x as the correct os.arch, it always returns the wrong platform "x86" instead.  Though we are applying the com.moowork.node of version v1.2.0, i think the problem is affecting all versions of this gradle plugin.

Linux on Z system:

os.arch = s390x
os.name = Linux
execute "uname -m" = s390x

execute "cat /proc/version":
Linux version 3.10.0-1160.36.2.el7.s390x (mockbuild@s390-037.build.eng.bos.redhat.com) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-44) (GCC) ) #1 SMP Wed Jul 7 22:55:15 EDT 2021

The node download url, eg, "https://nodejs.org/dist/v6.14.3/node-v6.14.3-linux-s390x.tar.gz", is expected. 

So can you help fix this issue?

**Comment by deepy** [comment]
This plugin is abandoned since 2019, but I've copied this issue to an actively maintained fork, I actually wanted to add specifically this support but was previously unable to find anyone who could provide me with `os.arch` and `uname`

**Comment by mengwan9** [comment]
@deepy yes, since we are from IBM, many projects are pushing into Linux on Z system and i noticed several projects are still using the plugin, so it seems this is a meaningful fix if you can do it.
